utils/sysfs/sysfs.go

```diff
@@ -37,9 +38,16 @@ const (
 	ppcDevTree   = "/proc/device-tree"
 	s390xDevTree = "/etc" // s390/s390x changes
 
-	coreIDFilePath    = "/topology/core_id"
-	packageIDFilePath = "/topology/physical_package_id"
-	meminfoFile       = "meminfo"
+	coreIDFilePath       = "/topology/core_id"
+	packageIDFilePath    = "/topology/physical_package_id"
+	meminfoFile          = "meminfo"
+	sysFsCPUTopology     = "topology"
+	CPUPhysicalPackageID = "physical_package_id"
```

Comment:
Would be great to leave some comments on what `physical_package_id` / `core_id` will represent, eg. xref: https://www.kernel.org/doc/Documentation/ABI/testing/sysfs-devices-system-cpu

```
core_id: the CPU core ID of cpu#. Typically it is the
		hardware platform's identifier (rather than the kernel's).
		The actual value is architecture and platform dependent.
```

```
	physical_package_id: physical package id of cpu#. Typically
		corresponds to a physical socket number, but the actual value
		is architecture and platform dependent.
```